specification/parameters/query-shipments-filter-carrier_status-category.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "filter[carrier_status][category]",
+  "in": "query",
+  "description": "Comma separated string of status categories to filter by. Only shipments with a status that corresponds with one of the categories will be in the response.",
```

Comment:
> Only shipments with a status that corresponds with one of the categories will be in the response.

Let's clarify here what "shipments with a status" means. Does it only check the current shipment status, or will it also return shipments that have that category in any of their historical shipment statuses?